### ISSUE TITLE
DF Chat Enhancements: Whisper selector

### ISSUE THREAD

**Issue by LukasPrism** [flamewave000/dragonflagon-fvtt#374]
Please consider adding a button which brings up a panel to select which player/s to whisper to. It's not always easy to remember whether you're supposed to use player name or character name, and what syntax to use for whispering to multiple recipients etc. Perhaps this could be integrated into the popup when you hit the / key but I'm not sure the best way to do that.
Thanks for a brilliant module!

**Comment by flamewave000** [comment]
I actually have though about the idea of improving the whisper interaction. I actually just tonight had a hard time getting the syntax correct so I could send a player a "quick" whisper lol

**Comment by LukasPrism** [comment]
Yeah I think a lot of people avoid whispering because it’s too difficult, and might actually use them more if it was easier. Monk’s Token Bar helps, but it should really be part of chat :)

**Comment by flamewave000** [comment]
Closing this as there is a great module that provides this functionality: [Chat Commander](https://foundryvtt.com/packages/_chatcommands)
